pyproject.toml

```diff
@@ -43,14 +43,18 @@ types-requests = "^2.28"
 types-urllib3 = "^1.26"
 pre-commit = "^2.21.0"
 commitizen = "^2.42.1"
-pdoc = "^12.3"
+sphinx = {version = "^7.2.6", python = ">=3.9,<4"}
+sphinx-rtd-theme = {version = "^2.0.0", python = ">=3.9,<4"}
 poethepoet = "^0.24.1"
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poe.tasks]
-test        = "poetry run pytest -m 'not ci_exempt and not flaky_test' tests/api/v2"
-flaky_test  = "poetry run pytest -m 'flaky_test' tests"
-lint        = "poetry run flake8 . --exclude 'jupyter' --count --select=E9,F63,F7,F82 --show-source --statistics"
\ No newline at end of file
+test        = "pytest -m 'not ci_exempt and not flaky_test' tests/api/v2"
+flaky_test  = "pytest -m 'flaky_test' tests"
+lint        = "flake8 . --exclude 'jupyter' --count --select=E9,F63,F7,F82 --show-source --statistics"
+docs-clean.shell  = "rm -r docs-source/auto; rm -r docs/*"
+docs-build.shell  = "sphinx-apidoc -e -f -d 5 -o docs-source/auto laceworksdk; sphinx-build -M html docs-source docs"
```

Comment:
do we need to handle the errors here? (seeing two commands)

Reply:
Good catch